vdr/src/client/client.rs

```diff
@@ -275,39 +249,84 @@ pub mod test {
             .to_string()
     }
 
-    fn contracts() -> Vec<ContractConfig> {
-        vec![
-            ContractConfig {
-                address: SCHEMA_REGISTRY_ADDRESS.to_string(),
-                spec_path: Some(build_contract_path(SCHEMA_REGISTRY_SPEC_PATH)),
-                spec: None,
-            },
-            ContractConfig {
-                address: CRED_DEF_REGISTRY_ADDRESS.to_string(),
-                spec_path: Some(build_contract_path(CRED_DEF_REGISTRY_SPEC_PATH)),
-                spec: None,
-            },
-            ContractConfig {
-                address: VALIDATOR_CONTROL_ADDRESS.to_string(),
-                spec_path: Some(build_contract_path(VALIDATOR_CONTROL_PATH)),
-                spec: None,
+    fn get_contract_conf(contract_name: &str) -> Option<Value> {
```

Comment:
please add contract type definition for simplicity.

Reply:
done